src/dev/konnector.md

```diff
@@ -2,35 +2,52 @@
 
 ## Introduction
 
-**New! Want an interactive tutorial on how to create a connector? Try [this one](https://tech.io/playgrounds/1482/cozy-connector-tutorial):**
+A connector is a simple script which imports data from other web services and put it in your cozy.
+Each connector in an independant application, managed by the [Cozy Collect](https://github.com/cozy/cozy-collect)
+application.
 
-A connector is a small script that allows to import data from an external website. Each connector is an independent application, managed by the Cozy Collect application.
+To protect your data, each connector runs inside a container in order to sandbox all their
+interactions with your data.
 
-To protect your data, each connector runs inside a sandbox where all their interactions with your data are under control.
+## How does it work ?
 
-From a technical point of view, connectors are Node.js applications executed inside a container. They communicate with the server using its API, like client side apps, and get an auth token every time they start. They need to register with a manifest, and ask permissions to the user.
+A connector is a NodeJS script. The target node version used to run your connector is the
+[current LTS version](https://nodejs.org) (8 at the moment).
 
-To ease the development of a connector, an npm package, named [cozy-connector-libs](https://github.com/cozy/cozy-konnector-libs) provides a lot of
-shared libraries. But you may need some other npm packages to help you running your connector: 
+Like client side apps, connectors communicate with the [Cozy Stack](https://cozy.github.io/cozy-stack/)
+using its API, and get an auth token every time they start. They need to register with a manifest,
+and ask permissions to the user.
 
- - [moment](http://momentjs.com/docs/) to manage dates 
+To ease the development of a connector, a npm package, named [cozy-konnector-libs](https://github.com/cozy/cozy-konnector-libs)
+provides some shared libraries which are adapted to be used for a connector :
+ - [cheerio](https://cheerio.js.org) to easily request html pages like jQuery
+ - [request-promise](https://github.com/request/request-promise): [request](https://github.com/request/request) wrapped in promises
+ - [request-debug](https://github.com/request/request-debug) which displays all the requests and
+   responses in the standard output. Check "debug" option in [requestFactory](https://github.com/cozy/cozy-konnector-libs/blob/master/docs/api.md#requestFactory)
 
-When the application is started, it also gets some data through environment variables:
+But you may need some other npm packages to help you running your connector and
+which are not integrated to cozy-konnector-libs :
```

Comment:
Je dirais "But you may need some other npm packages not integrated in cozy-konnector-libs to help you running your connector".

Reply:
s/integrated/included
d/But/
d/some/